lib/ex_crypto.ex

```diff
@@ -36,6 +36,33 @@ defmodule ExCrypto do
     {kind, Exception.normalize(kind, error), System.stacktrace}
   end
 
+  @aes_block_size 16
+  @iv_bit_length 128
+
+  defp key_and_iv_for_algorithm(raw_key, raw_iv, algorithm) do
+    bits = algorithm_bits(algorithm)
+    {iv, _} = bitstring_size(raw_iv, @iv_bit_length)
```

Comment:
@voltone on elixir slack pointed out a specific reason this behavior for IV in particular could be problematic, even in a bang method, if someone were using incrementing IVs, and we cut off the incrementing part... but it is a bang method...
had funny idea we could use LAST 128 bits, but that hurts interop, so dunno... maybe this bang method should only work on keys and still error/offer helper function for modifying IV to right length